scripts/check_ecosystem.py

```diff
@@ -58,13 +58,14 @@ async def clone(self: Self) -> "AsyncIterator[Path]":
 
 
 REPOSITORIES = {
-    "zulip": Repository("zulip", "zulip", "main"),
-    "bokeh": Repository("bokeh", "bokeh", "branch-3.2"),
+    "airflow": Repository("apache", "airflow", "main", select="ALL"),
+    "bokeh": Repository("bokeh", "bokeh", "branch-3.2", select="ALL"),
+    "cibuildwheel": Repository("pypa", "cibuildwheel", "main"),
+    "disnake": Repository("DisnakeDev", "disnake", "main"),
```

Comment:
Also adding `disnake`.

Reply:
This broke the check job entirely, as it's "master", not "main", in this check. :(

I think changes shouldn't fail jobs, but being completely broken should probably produce a red x on the job.